Sources/Subs.php

```diff
@@ -3274,93 +3274,66 @@ function template_header()
 
 	header('Content-Type: text/' . (isset($_REQUEST['xml']) ? 'xml' : 'html') . '; charset=UTF-8');
 
-	$checked_securityFiles = false;
-	$showed_banned = false;
-	foreach ($context['template_layers'] as $layer)
+	$show_warnings = empty($context['layout_loaded']) || $context['layout_loaded'] == 'default';
+	$show_warnings &= allowedTo('admin_forum');
+	$show_warnings &= empty($user_info['is_guest']);
+	if ($show_warnings)
 	{
-		// May seem contrived, but this is done in case the body and main layer aren't there...
```

Comment:
Does this situation ever occur?

Reply:
Not now, since we largely made these irrelevant with the new templates.